core/org.openjdk.jmc.flightrecorder.rules.jdk/src/main/resources/org/openjdk/jmc/flightrecorder/rules/jdk/messages/internal/messages.properties

```diff
@@ -241,7 +241,7 @@ FileReadRuleFactory_TEXT_NO_EVENTS=There are no file read events in this recordi
 # {0} is a time period
 FileReadRuleFactory_TEXT_WARN=There are long file read pauses in this recording (the longest is {0}).
 # {0} is a time period, {1} is a time stamp, {2} is a size in bytes
-FileReadRuleFactory_TEXT_WARN_LONG=The longest recorded file read took {0} to read {2} from {1}.
+FileReadRuleFactory_TEXT_WARN_LONG=The longest recorded file read took {0} to read {2} from {1}. Average time of all IO of the recording: {3}. Total time of all IO of the recording: {4}. Total time of all IO of the file {1}: {5}.
```

Comment:
Maybe good to add "recorded". Average time of the recorded I/O, Total time of the recorded I/O, Total time of the recorded I/O for the file. May help emphasize that it is whatever was recorded that we're calculating over. "All I/O" my prime people to think that all I/O would be recorded, which is usually not true - most I/O related events will normally be thresholded to only record outliers.